elmslie-android/src/main/java/vivid/money/elmslie/android/androidstore/AndroidStore.kt

```diff
@@ -0,0 +1,105 @@
+package vivid.money.elmslie.android.androidstore
+
+import android.os.Bundle
+import androidx.activity.ComponentActivity
+import androidx.annotation.MainThread
+import androidx.fragment.app.Fragment
+import androidx.lifecycle.AbstractSavedStateViewModelFactory
+import androidx.lifecycle.SavedStateHandle
+import androidx.lifecycle.ViewModel
+import androidx.lifecycle.ViewModelProvider
+import androidx.lifecycle.ViewModelStoreOwner
+import androidx.savedstate.SavedStateRegistryOwner
+import vivid.money.elmslie.core.store.Store
+
+@MainThread
+fun <
+    Event : Any,
+    Effect : Any,
+    State : Any,
+> androidStore(
```

Comment:
I can't think of usecase for this outside of fragment/activity, maybe internal, then?

Reply:
I still don't know how it will work with pure compose, that's why I've decided to make it public and conduct some experiments inside the project.

Reply:
I mean that it won't be returning lazy in compose for sure, so no need for it to be public

Reply:
and it doesn't provide any defaults  
it clearly looks like internal api

Reply:
but I can try to use this method inside our project to do some experiments with compose without MRs to this repo :)

it will be easy to make it internal after the alpha version

Reply:
ok, I will just copy all this to our project as our own package